src/unitxt/artifact.py

```diff
@@ -396,6 +409,53 @@ def verify_instance(
         return instance
 
 
+class ArtifactLink(Artifact):
+    # the artifact linked to, expressed by its catalog id
+    artifact_linked_to: str = NonPositionalField(
+        default=None, required=False, also_positional=False
```

Comment:
Shouldn't it be a required field without a default? Any use case gor not initializing it?

Reply:
Yes, agree..  just copied over without thinking. Fixed now..